README.md

```diff
@@ -1,2 +1,89 @@
 # vm-memory
-Virtual machine's guest memory crate
+A library to manage and access virtual machine's physical memory.
```

Comment:
This seems like an in depth description of the crate that would probably have a better place in a design document. I find short readmes with links to other documents easier to follow (it also looks prettier on crates.io).
The readme is the front page of the crate, so usually what I notice is that people have the following subsections in it:
- Summary
- Minimal Usage Example
- Contributions (optional)
- Links to in depth documentation
- License

Some examples of such crates:
- [backtrace](https://crates.io/crates/backtrace)
- [serde](https://crates.io/crates/serde)
- [lazy_static](https://crates.io/crates/lazy_static)

I find it easier to have a `Summary` at the beginning so the user can get a grasp on what the crate is and what is it useful for. It also worth having information in the readme from the user perspective. How can you use the vm-memory crate? What is it doing for you?